CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## [5.4.3]
+
+## Changed
+* Added `--vv_raster` as an input for `water_map` when producing flood depth estimation maps.
```

Comment:
```suggestion
- Upgraded to asf-tools [v0.4.2](https://github.com/ASFHyP3/asf-tools/blob/develop/CHANGELOG.md#41) from v0.4.1
```
I think we can just say this here.

We'll also need to update the `asf_tools` version in the dockerfile:
https://github.com/ASFHyP3/hyp3-gamma/blob/develop/Dockerfile#L3